tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -1558,6 +1558,20 @@ STATIC_ASSERT(__cpp_lib_string_contains == 202011L);
 #endif
 #endif
 
+#if _HAS_CXX23
+#ifndef __cpp_lib_string_resize_and_overwrite
+#error __cpp_lib_string_resize_and_overwrite is not defined
+#elif __cpp_lib_string_resize_and_overwrite != 202110L

```

Comment:
Mention P0980 in this file above

Reply:
What do you mean?

Reply:
Sorry  nothing. Mistaken this for yvals_core